src/integ/groovy/org/mastercoin/test/rpc/MSCSendToOwnersSpec.groovy

```diff
@@ -27,21 +27,32 @@ class MSCSendToOwnersSpec extends BaseRegTestSpec {
     def "STO calculates correct fees for the simple case"() {
         setup:
         def startingBTC = 10.0
-        def startingMSC = 1000
-        def amountSent = 100
+        def startingMSC = 1000.0
+        def amountSent = 100.0
         def fundedAddress = createFundedAddress(startingBTC, startingMSC)
         def currencyID = TMSC
+        def expectedBalance = 0.0
 
         when: "We Send to Owners"
-        def startingPropBal = getbalance_MP(fundedAddress, currencyID).balance
+        def startBalances = getallbalancesforid_MP(currencyID)
+        def startBalanceSender = getbalance_MP(fundedAddress, currencyID).balance
+        def numberOfHolders = startBalances.size()
         sendToOwnersMP(fundedAddress, currencyID, amountSent)
 
-        and: "we generate a block"
+        and: "We generate a block"
         generateBlock()
-        def balances = getallbalancesforid_MP(currencyID)
-        def numberDest = balances.size() - 1
-        def totalFee = numberDest * stoFeePerAddress
-        def expectedBalance = (numberDest == 0) ? startingPropBal : startingPropBal - amountSent - totalFee
+
+        // The fee for each receiver is #stoFeePerAddress
+        if (numberOfHolders > 1) {
```

Comment:
Sure, I think this would be beneficial, but probably in form of a different PR. The intend of this one was not necessarily to change the logic, but fix the issue of test failures for the two cases:

- There are many holders and the amount sent to the owners doesn't "reach" everyone. The unreached addresses create a failure. This is why I introduced the "number of owners" and "balances changed".
- At the time of test start there is no holder other than the one we create. In this case the test fails, because the sender's balance doesn't change at all. It's addressed in MSCSendToOwnersSpec, but not yet in MSCSendToOwnersConsensusComparisonSpec.